soundbay/inference.py

```diff
@@ -134,8 +134,45 @@ def inference_to_file(
     output_file = output_path / filename
     concat_dataset.to_csv(index=False, path_or_buf=output_file)
 
+    # save raven file
```

Comment:
Looks good,
We should add this to the hydra configuration .
(True/ False for saving it as raven file

Reply:
good idea, will do